Atomic/syscontainers.py

```diff
@@ -861,7 +861,7 @@ def _do_checkout(self, repo, name, img, upgrade, deployment, values, destination
     def _get_preinstalled_containers_path(self):
         return ATOMIC_USR
 
-    def _get_system_checkout_path(self):
+    def get_system_checkout_path(self):
```

Comment:
One of the previous iterations of my code used this, can revert to be private again.